bundler/lib/dependabot/bundler/file_fetcher/require_relative_finder.rb

```diff
@@ -10,7 +10,7 @@
 module Dependabot
   module Bundler
     class FileFetcher
-      # Finds the paths of any files included using `require_relative` in the
+      # Finds the paths of any files included using `require_relative` and `eval` in the
```

Comment:
😍 keeping the code comment relevant.

Should the file be renamed slightly since it's currently scoped to `require_relative_finder`?

Reply:
I change it as `IncludePathFinder` to make support upcoming implementations such as `require` and `load` as well.